src/basic/in-addr-util.c

```diff
@@ -464,3 +464,45 @@ int in_addr_mask(int family, union in_addr_union *addr, unsigned char prefixlen)
 
         return -EAFNOSUPPORT;
 }
+
+int in_addr_prefix_from_string(const char *p, int family, union in_addr_union *prefix, uint8_t *prefixlen) {
```

Comment:
while we don't follow this rule everywhere it's a good idea to prefix return parameters with "ret_". i.e. the parameter "prefix" would better be called "ret_prefix", and "prefixlen" similar